core/fragment.c

```diff
@@ -6819,8 +6821,10 @@ flush_fragments_and_remove_region(dcontext_t *dcontext, app_pc base, size_t size
                                   bool own_initexit_lock, bool free_futures)
 {
     flush_fragments_in_region_start(dcontext, base, size, own_initexit_lock, free_futures,
-                                    true /*exec invalid*/,
-                                    false /*don't force synchall*/ _IF_DGCDIAG(NULL));
+                                    true /*exec invalid*/, false /*don't force synchall*/,
+                                    THREAD_SYNCH_NO_LOCKS_NO_XFER
+
```

Comment:
format nit: remove extra vertical spacing. Surprised that clang didn't complain.